src/e2e/multiple-files.test.ts

```diff
@@ -3,60 +3,56 @@
  *
  * SPDX-License-Identifier: MIT
  */
-import { App, LoadDialogType } from "./app";
+import { expect } from "@playwright/test";
+import { LoadDialogType } from "./app.js";
+import { test } from "./app-test-fixtures.js";
 
-describe("multiple-files", () => {
-  const app = new App();
-  beforeEach(app.reset.bind(app));
-  afterEach(app.screenshot.bind(app));
-  afterAll(app.dispose.bind(app));
-
-  it("Copes with hex with no Python files", async () => {
+test.describe("multiple-files", () => {
+  test("Copes with hex with no Python files", async ({ app }) => {
     // Probably best for this to be an error or else we
     // need to cope with no Python at all to display.
     await app.loadFiles("src/micropython/main/microbit-micropython-v2.hex");
-
-    await app.findAlertText(
+    await app.expectAlertText(
       "Cannot load file",
       "No appended code found in the hex file"
     );
   });
 
-  it("Add a new file", async () => {
+  test("Add a new file", async ({ app }) => {
     await app.createNewFile("test");
 
-    await app.findVisibleEditorContents(/Your new file/);
-    await app.findProjectFiles(["main.py", "test.py"]);
+    await app.expectEditorContainText(/Your new file/);
+    await app.expectProjectFiles(["main.py", "test.py"]);
   });
 
-  it("Prevents deleting main.py", async () => {
-    expect(await app.canDeleteFile("main.py")).toEqual(false);
+  test("Prevents deleting main.py", async ({ app }) => {
+    expect(await app.isDeleteFileOptionDisabled("main.py")).toEqual(true);
   });
 
-  it("Copes with non-main file being updated", async () => {
+  test("Copes with non-main file being updated", async ({ app }) => {
     await app.loadFiles("testData/usermodule.py", {
       acceptDialog: LoadDialogType.CONFIRM_BUT_LOAD_AS_MODULE,
     });
-    await app.switchToEditing("usermodule.py");
-    await app.findVisibleEditorContents(/b_works/);
+    await app.editFile("usermodule.py");
+    await app.expectEditorContainText(/b_works/);
 
     await app.loadFiles("testData/updated/usermodule.py", {
       acceptDialog: LoadDialogType.CONFIRM_BUT_LOAD_AS_MODULE,
     });
 
-    await app.findVisibleEditorContents(/c_works/);
+    await app.expectEditorContainText(/c_works/);
   });
 
-  it("Shows warning for third-party module", async () => {
+  test("Shows warning for third-party module", async ({ app }) => {
     await app.loadFiles("testData/module.py", {
       acceptDialog: LoadDialogType.CONFIRM,
     });
-    await app.switchToEditing("module.py");
+    await app.editFile("module.py");
     await app.findThirdPartyModuleWarning("a", "1.0.0");
```

Comment:
This seems like the only thing still named findXXX. Fix for consistency?

Reply:
179e918954ddcdffc85aaf2646eaf09c354e6dd3